src/pods/edit-table/edit-table.component.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
     value: FieldVm[K]
   ) => void;
   onDeleteField: (fieldId: GUID) => void;
-  onAddField: (fieldId: GUID, isChildren: boolean) => void;
+  onAddField: (fieldId: GUID, isChildren: boolean, newFieldId: GUID) => void;
```

Comment:
4 params too mucho let's simplify it (we may use same interface as before)

```ts
interface OriginFieldInfo {
  fieldId: GUID; 
  isChildren: boolean;
}
```


then

```diff
- onAddField: (fieldId: GUID, isChildren: boolean, newFieldId: GUID) => void;
+ onAddField: (originFieldInfo : OriginFieldInfo, newFieldId: GUID) => void;
```